### PR TITLE
added the need for settings.init.force_start.set(true)

### PR DESCRIPTION
**Pull request by theotherjenkutler**

### CODE REVIEW

File: doc/content/dynamic_sources.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ First some outlines:
 
 * This example is meant to create a new source and outputs. It is not easy currently to change a source being streamed
 * The idea is to create a new output using a telnet/server command.
+* In order for a liquidsoap script to run without an active sort at startup, it is necessary to include settings.init.force_start.set(true) at the start of the script.
```
Comment (rrolla) [inline doc/content/dynamic_sources.md:9]:
```suggestion
* In order for a liquidsoap script to run without an active source at startup, it is necessary to include settings.init.force_start.set(true) at the start of the script. 
```

**Review by toots — APPROVED** [review]
That looks good to me! @smimram ?

Thanks for the PR!

File: doc/content/dynamic_sources.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ First some outlines:
 
 * This example is meant to create a new source and outputs. It is not easy currently to change a source being streamed
 * The idea is to create a new output using a telnet/server command.
+* In order for a liquidsoap script to run without an active source at startup, it is necessary to include settings.init.force_start.set(true) at the start of the script.
```
Comment (smimram) [inline doc/content/dynamic_sources.md:9]:
```suggestion
* In order for a liquidsoap script to run without an active source at startup, it is necessary to include `settings.init.force_start.set(true)` at the start of the script. 
```

**Comment by toots** [comment]
Thanks!

**Pull request merged**